packages/web-runtime/src/store/user.js

```diff
@@ -99,6 +101,14 @@ const actions = {
         const userGroups = await client.users.getUserGroups(login.id)
         const user = await client.users.getUser(login.id)
 
+        // FIXME: Can be removed as soon as the uuid is integrated in the OCS api
+        const graphClient = clientService.graphAuthenticated(instance, token)
+
+        let graphUser
+        if (context.rootState.user.version.edition === 'reva') {
```

Comment:
That's a no-no. We don't want to differentiate backends. Either you can determine if the graph api is available (e.g. `graphClient` could have an `available()` getter which makes a ping (basically any request) via graph API, if that fails we remember in a variable that the graphClient is not available. Or you find some other attribute how to check if the graph API is available. E.g. introduce a capability in ocis.

Reply:
Ahh okay, I though this is legit as we have several `isOcis`-checks floating around. I'll discuss with the backend team what's most suitable here. A capability sounds nice.

Reply:
As discussed in chat: We can use `capabilities.spaces.enabled` for now.

Reply:
But I like the idea with the `available()` getter. Let me introduce that one at least.

Reply:
The existing `isOcis` checks are deprecated and will be removed soonish. :-)

`available()` getter which wraps the capabilities check is superb 👍 then it's easy to switch over to proper `graph ` permissions later on.

Reply:
I've created https://github.com/owncloud/web/issues/6551 to keep track on this. I think this PR is fine without for now.